src/app/ipv/router.js

```diff
@@ -38,6 +41,43 @@ router.post(
   (req, res, next) =>
     handleUpdateNameDobAction(req, res, next, "update-name-date-birth"),
 );
+router.post(
+  "/page/page-multiple-doc-check",
+  parseForm,
+  csrfProtection,
+  formRadioButtonChecked,
+  (req, res, next) =>
+    handleMultipleDocCheck(req, res, next, "page-multiple-doc-check"),
+);
+router.post(
+  "/page/pyi-cri-escape",
+  parseForm,
+  csrfProtection,
+  formRadioButtonChecked,
+  (req, res, next) => handleEscapeAction(req, res, next, "pyi-cri-escape"),
+);
+router.post(
+  "/page/pyi-suggest-other-options",
+  parseForm,
+  csrfProtection,
+  formRadioButtonChecked,
+  (req, res, next) =>
+    handleEscapeAction(req, res, next, "pyi-suggest-other-options"),
```

Comment:
The page `pyi-suggest-other-options` existed on states which did't use the "end" journey event